README.md

```diff
@@ -26,7 +26,7 @@ Please check out our page on [Jetpack Compose for Microsoft Surface Duo](https:/
 
 ## Prerequisites
 
-- Jetpack Compose version: `1.0.4`
+- Jetpack Compose version: `1.0.5`
 
 - AndroidX WindowManager version: `1.0.0-beta02`
```

Comment:
This needs to be updated to `beta03` and probably renamed to `Jetpack WindowManager`.

Reply:
good catch!